pkg/standalone/client.go

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+type DaprProcess interface {
+	List() ([]ListOutput, error)
+}
+
+type daprProcess struct {
+}
+
+// Client is the interface the wraps all the methods exposed by the Dapr CLI.
+type Client interface {
+	// Get is used to invoke a method on a Dapr application with GET verb.
+	Get(appID, method string) (string, error)
```

Comment:
Get and Post seem vague to me. InvokeServicePost or InvokeServiceGet would be explicit. 

cc/ @tcnghia

Reply:
Made it InvokeGet and InvokePost... as we do not have any other get or post methods ... also it is not direct post or get but service invocation ... so Invoke makes sense ...